web/src/components/icon/index.tsx

```diff
@@ -7,12 +7,13 @@ import * as icons from '../../icons'
 type IconSize = 'small' | 'medium' | 'large'
 
 export type IconTypes = 'arrowLeft' | 'close' | 'people' | 'person'
-type GenericIconProps = Omit<DOMProps & SVGAttributes<SVGElement>, 'name'> & {
+export type GenericIconProps = Omit<DOMProps, 'name'> & {
```

Comment:
Might be `IconProps` instead `GenericIconProps`?

Reply:
true, was some old code that I had. thanks!

Reply:
fixed in b532f29240724a50183701ae024affcc799bc705